portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java

```diff
@@ -48,26 +51,42 @@
   @Override 
   public void doPost(HttpServletRequest request, HttpServletResponse response)
   throws IOException {
+    
+    Enumeration<String> paramNames = request.getParameterNames();
+    // for (String myVar : paramNames) {
+    //   System.err.println(myVar);
+    // }
+
+    while(paramNames.hasMoreElements()) {
+      System.err.println(paramNames.nextElement());
+    }
 
     // Receive input from the create a comment form
     int blogNumber;
     try {
       blogNumber = ValidateInput.getUserNum(request, "blog-number", 1, 
           CommentConstants.MAX_NUM_BLOGS);
     } catch (Exception e) {
-      System.err.println("Please enter an integer between 1 to " + 
-          CommentConstants.MAX_NUM_BLOGS + ".");
+      String errorMessage = e.getMessage();
+      System.err.println(errorMessage);
+
+      String jsonErrorMessage = new Gson().toJson(errorMessage);
+      response.setContentType("application/json;");
+      response.getWriter().println(jsonErrorMessage);
```

Comment:
This operation is done multiple times- instead of duplicating code, can you move this code into a method and call it from each place?